hack/kind-up.sh

```diff
@@ -122,6 +122,7 @@ setup_containerd_registry_mirrors() {
     setup_containerd_registry_mirror $NODE "registry.k8s.io" "https://registry.k8s.io" "http://${REGISTRY_HOSTNAME}:5006"
     setup_containerd_registry_mirror $NODE "quay.io" "https://quay.io" "http://${REGISTRY_HOSTNAME}:5007"
     setup_containerd_registry_mirror $NODE "europe-docker.pkg.dev" "https://europe-docker.pkg.dev" "http://${REGISTRY_HOSTNAME}:5008"
+    setup_containerd_registry_mirror $NODE "garden.local.gardener.cloud:5001" "http://garden.local.gardener.cloud:5001" "http://${REGISTRY_HOSTNAME}:5001"
```

Comment:
Don't you need this config also for a Shoot Node? Right not it is only applied for the kind cluster's Node.
Shoot Node ones are applied in https://github.com/gardener/gardener/blob/3b61cb9f7a227259f5d0034e90fb2dcebc664152/pkg/provider-local/webhook/controlplane/ensurer.go#L81-L89.

Reply:
As far as I can tell from looking at the code at: https://github.com/gardener/gardener/blob/cc0854ffc7c6100f0679a67b7866d5cf6602f5b5/hack/kind-up.sh#L115-L127

This change is done for every node that is created by kind. Wouldn't that include the Shoot node?

Reply:
I think @ialidzhikov is right. If you have workload on the shoot node (does Lakom run there as well?), then you need to also adjust [gardener/pkg/provider-local/webhook/controlplane/ensurer.go](https://github.com/gardener/gardener/blob/3b61cb9f7a227259f5d0034e90fb2dcebc664152/pkg/provider-local/webhook/controlplane/ensurer.go#L81-L89), since the machine has it's own containerd daemon.

Reply:
Lakom does not actually run on the Shoot node. It only installs webhooks on it, while the actual admission controllers live on the Seed.
But I'm still not sure whether what I wrote in the previous comment is correct. I'll test it and report back

Reply:
I would vote to add the same config to the shoot nodes for completeness, in the future lakom or some other component might need it.

Reply:
It's a simple change, so I'll add it